core/types/tx_blob.go

```diff
@@ -85,6 +86,14 @@ func (sc *BlobTxSidecar) encodedSize() uint64 {
 	return rlp.ListSize(blobs) + rlp.ListSize(commitments) + rlp.ListSize(proofs)
 }
 
+type BlobTxSidecarResp struct {
```

Comment:
why not just use the `BlobSidecar`: https://github.com/bnb-chain/bsc/blob/master/core/types/blob_sidecar.go#L23